### ISSUE TITLE
Error reading rule files: 0500-owncloud_rules.xml. Error: mismatched tag: line 46, column 39"

### ISSUE THREAD

**Issue by kulozzzz** [wazuh/wazuh#606]
HI:
Today, after I upgraded my rules, I used an api to query my rules for errors:
```
{
    "error": 1201,
    "message": "Error reading rule files: 0500-owncloud_rules.xml. Error: mismatched tag: line 46, column 39"
}
```
Do not know if this xml file has any problems?

**Comment by mgmacias95** [comment]
Hello @kulozzzz ,

This is an error in our internal XML parser, which has been fixed in https://github.com/wazuh/wazuh/pull/484.

The error is caused because our internal parser was not able to read `@` in XML attributes. As you can see, the line 46 of _owncloud_ rules, has this bug:
https://github.com/wazuh/wazuh/blob/353f5cdaf2d5c90388acb000bf5f7933c3e8ceb6/etc/rules/0500-owncloud_rules.xml#L43-L48

Which wazuh version are you using?

Best regards,
Marta

**Comment by kulozzzz** [comment]
HI ，@mgmacias95 Wazuh server The version we use is v 3.2.1, wazah api is v 3.2.2

**Comment by mgmacias95** [comment]
Hello @kulozzzz,

The fix has been added in Wazuh v3.2.2 (our latest release). Try updating your installation and check if the bug is fixed.

Best regards,
Marta

**Comment by kulozzzz** [comment]
@mgmacias95 Ok, I try to update it to see if it solves the problem.
thank!
